packages/grid-pro/src/lit/column-renderer-directives.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2022 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { directive } from 'lit/directive.js';
+import { microTask } from '@vaadin/component-base/src/async';
+import { Debouncer } from '@vaadin/component-base/src/debounce';
```

Comment:
Let's use `.js` extensions for consistency:

```suggestion
import { microTask } from '@vaadin/component-base/src/async.js';
import { Debouncer } from '@vaadin/component-base/src/debounce.js';
```